doc/features.rst

```diff
@@ -17,7 +17,7 @@ ________
   
 * Supports a truly reproducible workflow by minimizing the code needed for analysis and visualization.
 * Already used in a variety of research projects, from conception to final publication.
-* All HoloViews objects can be pickled and unpickled.
+* All HoloViews objects can be pickled and unpickled, with no plotting-library dependencies.
```

Comment:
Ok. So if you use a matplotlib object in style options, you will obviously have trouble unpickling the style if matplotlib is not available. This is a good reason to avoid using matplotlib objects in this way...